src/settings.py

```diff
@@ -13,11 +13,10 @@
     cast=lambda filename: tomllib.loads(Path(filename).read_text()),
     default={},
 )
+PLUGINS = config("PLUGINS", cast=lambda x: x.split(",") if isinstance(x, str) else x, default=[])
```

Comment:
decouple's csv helper solves this problem 

https://github.com/HBNetwork/python-decouple?tab=readme-ov-file#built-in-csv-helper